index.mjs

```diff
@@ -57,6 +57,15 @@ import yaml from 'js-yaml';
  *   built-ins/Temporal/Duration/compare/argument-string-negative-fractional-units.js
  *   built-ins/Temporal/Duration/from/argument-string-negative-fractional-units.js
  *   ```
+ * @property {number|string=2000} timeoutMsecs Optional number of milliseconds
```

Comment:
? I don't think this is needed / is duplicate with the block in the options bag?

Reply:
Sadly, it's needed. If I remember correctly, VS Code doesn't expand JSDoc typedefs. So if you want to show JSDoc info on the function name, the parameter list, and for individual variables in the options object, then you need to clone in both places. Let me know if you want me to dig up the VS Code bug report asking for this to be changed.

Reply:
No - that's ok. I didn't know that.

Maybe in the future we should migrate this repo to TS to maybe avoid this problem? :) It might also be nice to vend some typings for this.

Reply:
PRs welcome! :-) I'm not sure porting the implementation to TS would justify the added complexity for one small file of implementation code. But adding a .d.ts could be nice.